src/components/Text.tsx

```diff
@@ -0,0 +1,184 @@
+import {
+  TGradientColorStops,
+  TTextColor,
+  backgroundClip,
+  backgroundImage,
+  classnames,
+  dropShadow,
+  fontFamily,
+  fontSize,
+  fontWeight,
+  gradientColorStops,
+  letterSpacing,
+  lineHeight,
+  textAlign,
+  textColor,
+  textOverflow,
+  textTransform,
+} from 'classnames/tailwind'
+import ChildrenProp from 'models/ChildrenProp'
+import classNamesToString from 'helpers/classNamesToString'
+
+const foldText = (
+  color?: TTextColor,
+  gradientFrom?: TGradientColorStops,
+  gradientTo?: TGradientColorStops
+) =>
+  classnames(
+    fontFamily('font-primary'),
+    fontSize('text-2xl', 'md:text-3xl'),
+    lineHeight('leading-8', 'md:leading-10'),
+    fontWeight('font-bold'),
+    textColor(
+      gradientFrom && gradientTo ? 'text-transparent' : color || 'text-accent'
+    ),
+    backgroundImage(
+      gradientFrom && gradientTo ? 'bg-gradient-to-r' : undefined
+    ),
+    backgroundClip(gradientFrom && gradientTo ? 'bg-clip-text' : undefined),
+    gradientColorStops(gradientFrom, gradientTo)
+  )
+export function FoldText({
+  color,
+  gradientFrom,
+  gradientTo,
+  children,
+}: ChildrenProp & {
+  color?: TTextColor
+  gradientFrom?: TGradientColorStops
```

Comment:
How many gradients do we actually have? We should limit this argument (maybe create a type `FoldTextGradients = 'accent-to-primary`, for example), please note, that tailwind applies `transparent` automatically if you didn't provide `from` or `to`. Nikita already asked me about such a thing previously

Reply:
Make gradient static